modules/git/repo_branch_nogogit.go

```diff
@@ -22,13 +22,13 @@ func (repo *Repository) IsBranchExist(name string) bool {
 }
 
 // GetBranches returns all branches of the repository.
-func (repo *Repository) GetBranches() ([]string, error) {
-	return callShowRef(repo.Path, BranchPrefix, "--heads")
+// if limit = 0 it will not limit
```

Comment:
GetBranches returns branches from the repository, skipping skip initial branches and returning at most limit branches, or all branches if limit is 0.